### ISSUE TITLE
pquery: inconsistent maintainer output

### ISSUE THREAD

**Issue by ajakk** [pkgcore/pkgcore#281]
For example, `net-misc/hylafaxplus` and `app-misc/physlock` both have a `<!--maintainer-needed-->` tag in their `metadata.xml`, but running `pquery --blame` and `pquery --one-attr maintainers` produces incongruent output:

```sh
~ $ pquery --blame net-misc/hylafaxplus
~ $ pquery --blame app-misc/physlock
app-misc/physlock-11-r1 maintainers=""
~ $ pquery --one-attr maintainers app-misc/physlock

~ $ pquery --one-attr maintainers net-misc/hylafaxplus
```

Calling `pquery` for `hylafaxplus` doesn't produce any output, where it does for `physlock`. HEAD of my repo is at `dfae9410d6a0d2ec1f284a1fd74a31d534ffebf9` of the sync-able git repo. Notably, `hylafaxplus` has two ebuild versions and a FILESDIR where `physlock` has neither.

**Comment by radhermit** [comment]
Here's what I get:

```sh
~ $ pquery --blame net-misc/hylafaxplus
net-misc/hylafaxplus-5.5.5 maintainers=""
net-misc/hylafaxplus-5.6.1 maintainers=""
~ $ pquery --blame app-misc/physlock
app-misc/physlock-11-r1 maintainers=""
~ $ pquery --one-attr maintainers net-misc/hylafaxplus


~ $ pquery --one-attr maintainers app-misc/physlock

```

Everything looks fine to me. The --blame option shows that all versions don't have maintainers and the --one-attr usage appears to output a blank line per version signifying that each version doesn't have a maintainer. Whether that's the best output to use in the case where no maintainer exists is a different issue.

**Comment by ajakk** [comment]
Ok, I'll just assume something is wrong for me locally.

**Comment by ajakk** [comment]
This seems to also be happening here:

https://qa-reports.gentoo.org/output/maintainer-needed.html

Which is generated by: https://gitweb.gentoo.org/proj/qa-scripts.git/tree/maintainer-needed.sh

**Comment by ferringb** [comment]
@ajakk please provide a raw repro against gentoo repo output.  I'm assuming one exists; my ask is "give me something I can fix".

**Comment by ajakk** [comment]
Hm, I'm not really sure what I meant in my previous comment. In any case, I do still see incongruent output in `pquery --blame` against different maintainer-needed packages:

```
~ $ pquery --blame app-doc/cppman
app-doc/cppman-0.5.3 maintainers=""
app-doc/cppman-0.5.4 maintainers=""
~ $ pquery --blame net-misc/hylafaxplus
~ $
```

**Comment by ferringb** [comment]
I'm pretty sure the problem here is a license filter:
```
ferringb@frith ~/gentoo/pkgcore $ pquery hylafaxplus app-doc/cppman --raw --attr license --blame
app-doc/cppman-0.5.3 license="GPL-3" maintainers=""
app-doc/cppman-0.5.4 license="GPL-3" maintainers=""
net-misc/hylafaxplus-7.0.6-r3 license="hylafaxplus" maintainers=""
```

afaik, we don't have a good `--attr` option for "show me the license that matched, but your output is pretty clear that hylafaxfplus is not 'visible'- which means it's either filter by keywords or license.

Please close this if it my assumption is correct, but feel free to open a ticket w/ asks- frankly suggestions- about how to make pquery clearer in this particular case.

**Comment by ajakk** [comment]
Yeah, you're right, it seems like the discrepancy is related to visbility due to license. Are you asking me to open a new ticket for the confusing behavior I described in my original comment?
